### ISSUE TITLE
Pytest needs `atomicwrites`

### ISSUE THREAD

**Issue by crocojim18** [pgh-public-meetings/city-scrapers-pitt#102]
In trying to run `pytest`, I got the following error
`Traceback (most recent call last):
  File "C:\Users\Marty\AppData\Local\Programs\Python\Python38\Lib\runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Marty\AppData\Local\Programs\Python\Python38\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Marty\.virtualenvs\city-scrapers-pitt-Gw6_Nbil\Scripts\pytest.exe\__main__.py", line 4, in <module>
  File "c:\users\marty\.virtualenvs\city-scrapers-pitt-gw6_nbil\lib\site-packages\pytest\__init__.py", line 6, in <module>
    from _pytest.assertion import register_assert_rewrite
  File "c:\users\marty\.virtualenvs\city-scrapers-pitt-gw6_nbil\lib\site-packages\_pytest\assertion\__init__.py", line 7, in <module>
    from _pytest.assertion import rewrite
  File "c:\users\marty\.virtualenvs\city-scrapers-pitt-gw6_nbil\lib\site-packages\_pytest\assertion\rewrite.py", line 272, in <module>
    from atomicwrites import atomic_write
ModuleNotFoundError: No module named 'atomicwrites'`

**Comment by bonfirefan** [comment]
Replicated with python 3.7:
```
$ pytest
Traceback (most recent call last):
  File "C:\Users\Bonnie\AppData\Local\Programs\Python\Python37\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\Bonnie\AppData\Local\Programs\Python\Python37\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Bonnie\Envs\city-scrapers-pitt-NdSyTKtS\Scripts\pytest.exe\__main__.py", line 4, in <module>
  File "c:\users\bonnie\envs\city-scrapers-pitt-ndsytkts\lib\site-packages\pytest\__init__.py", line 6, in <module>
    from _pytest.assertion import register_assert_rewrite
  File "c:\users\bonnie\envs\city-scrapers-pitt-ndsytkts\lib\site-packages\_pytest\assertion\__init__.py", line 7, in <module>
    from _pytest.assertion import rewrite
  File "c:\users\bonnie\envs\city-scrapers-pitt-ndsytkts\lib\site-packages\_pytest\assertion\rewrite.py", line 272, in <module>
    from atomicwrites import atomic_write
ModuleNotFoundError: No module named 'atomicwrites'
```

**Comment by bonfirefan** [comment]
We need to update to the latest version of pytest. For anyone with an old version of the project, run the following:
```
pipenv uninstall pytest
pipenv install pytest
```
I'll open an issue to change the pytest version in our pipfile as well

**Comment by ben-nathanson** [comment]
FIxed by #130
